src/NexusMods.App.UI/Pages/TextEdit/TextEditorPageView.axaml

```diff
@@ -19,12 +19,13 @@
 
                 <Line/>
 
-                <ComboBox x:Name="ThemeSelector" Classes="Tertiary">
+                <ComboBox x:Name="ThemeSelector" Classes="Tertiary" VerticalAlignment="Center" Height="32" >
                     <ComboBox.ItemTemplate>
                         <DataTemplate DataType="{x:Type grammars:ThemeName}">
-                            <ComboBoxItem>
-                                <TextBlock Text="{CompiledBinding}"/>
-                            </ComboBoxItem>
+                            <StackPanel Orientation="Horizontal" Classes="Spacing-1">
+                                <icons:UnifiedIcon Classes="ColorLens" Size="20"/>
+                                <TextBlock Text="{CompiledBinding}" Classes="BodyMDNormal"/>
+                            </StackPanel>
```

Comment:
I don't know if you noticed in my recent PRs, but I've stopped using classes:

```xaml
<StackPanel Orientation="Horizontal" Spacing="{StaticResource Spacing-1}">
  <icons:UnifiedIcon Value={x:Static icons:IconValues.ColorLens} Size="20"/>
  <TextBlock Text="{CompiledBinding}" Theme="{StaticResource BodyMDNormalTheme}" />
</StackPanel>
```

IDE at least doesn't complain about this, and go-to-definition works with static resources

Reply:
I have been trying to limit the usage of direct resource references outside the Theme project. Setting values directly on the controls has some problems with priority, making it very hard to overrule background color in pseudo class states, since assignments on the control directly always take precedence.

If we can't get IDE to recognize Classes we can reconsider this, but for now I'll stay consistent with how we have been doing it so far.